kubescape.yaml

```diff
@@ -9,25 +9,33 @@ package:
 environment:
   contents:
     packages:
+      - build-base
       - busybox
       - ca-certificates-bundle
-      - build-base
-      - python3
-      - go
-      - git
       - cmake
+      - git
+      - go
+      - python3
 
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/kubescape/kubescape
       tag: v${{package.version}}
-      expected-commit: e4110837c70807de7c37cca69f422f5951249d64
+      expected-commit: 3e2314a269f650f89e9bbdae590f02ea361fee0c
       destination: kubescape
 
   - runs: |
       cd kubescape
       export CGO_ENABLED=1
+      # Handle CVE-2022-41723 CVE-2023-39325 and CVE-2023-3978
+      go get golang.org/x/net@v0.17.0
+      go mod tidy
+      pushd git2go
+      go get golang.org/x/net@v0.17.0
+      go mod tidy
+      go mod vendor
```

Comment:
You may need to initialize the git submodule first: https://github.com/kubescape/kubescape/blob/master/Makefile#L10

Although, ideally this is a case where we'd extract https://github.com/libgit2/git2go to its own wolfi package